samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,393 @@
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to update and boot the nRF54H20 System-on-Chip (SoC) using the Software Update for Internet of Things (SUIT) procedure on the Application and Radio Cores of the SoC.
+The update on the nRF54H20 SoC can be done over Bluetooth® Low Energy or UART.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+You need the nRF Connect Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+  (The minimum required version is v1.9.)
+
+* `nRF Device Manager mobile app for iOS`_
+  (The minimum required version is v1.5.)
+
+For a SUIT update over UART, you need to install MCUmgr.
```

Comment:
If I'm not mistaken, it will be replaced bu nrfutil.

Reply:
Okay, please let me know/or we can discuss with team how this should be rewritten.